src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -124,63 +120,81 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
   }
 
   // Toggle group
-  .pf-m-toggle-group {
+  &.pf-m-toggle-group {
     --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__group--m-toggle-group--spacer);
+
+    > * {
+      display: none;
+      visibility: hidden;
+    }
+
+    .pf-c-data-toolbar__toggle {
+      display: inline-block;
+      visibility: visible;
+    }
+  }
+}
+
+// Item
+.pf-c-data-toolbar__item {
+  // Separator
+  &.pf-m-separator {
+    --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__item--m-separator--spacer);
+
+    align-self: center;
+    width: var(--pf-c-data-toolbar__item--m-separator--Width);
+    height: var(--pf-c-data-toolbar__item--m-separator--Height);
+    background-color: var(--pf-c-data-toolbar__item--m-separator--BackgroundColor);
   }
 
   // Overflow menu
-  .pf-m-overflow-menu {
+  &.pf-m-overflow-menu {
     --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__item--m-overflow-menu--spacer);
   }
 
   // Bulk select
-  .pf-m-bulk-select {
+  &.pf-m-bulk-select {
     --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__item--m-bulk-select--spacer);
   }
 
   // Search filter
-  .pf-m-search-filter {
+  &.pf-m-search-filter {
     --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__item--m-search-filter--spacer);
   }
 
   // Search filter
-  .pf-m-chip-group {
+  &.pf-m-chip-group {
     --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__item--m-chip-group--spacer);
   }
 
   // Label
-  .pf-m-label {
+  &.pf-m-label {
     --pf-c-data-toolbar--spacer: var(--pf-c-data-toolbar__item--m-label--spacer);
 
     font-weight: var(--pf-c-data-toolbar__item--m-label--FontWeight);
   }
 
   // Total items
-  .pf-m-pagination {
+  &.pf-m-pagination {
     margin-left: auto;
 
     .pf-c-pagination {
       flex-wrap: nowrap;
     }
   }
 
-  .pf-m-clear {
+  &.pf-m-clear {
     justify-self: end;
   }
 
   // If pagination is sibling of align-right
-  .pf-m-align-right ~ * {
+  &.pf-m-align-right ~ * {
     margin-left: 0;
   }
 }
 
-.pf-c-data-toolbar__content,
-.pf-c-data-toolbar__group {
-  display: flex;
-  align-items: center;
-}
-
-// Group
+// __group and __item spacer values have to be defined after base styling and modifiers
+/* stylelint-disable no-duplicate-selectors */
```

Comment:
Why is this the case? I'm probably missing it, but I don't see why you couldn't move these up into their respective selectors above.